atlp-rwanda/Team-Sostene-E-commerce-bn
# #184651063  implement two factor authentication feature

### What does this PR do?
* Create TFA that acts as extra feature to provide more security to sellers
### Description of Task to be completed?
- Implement two factor authentication to admin roles(sellers)
- Ensure that only registered seller is the one receives that feature.
- Allow the final process of logging to the website to be done after TFA process is done

### How should this be manually tested?
- After registering user might have the option for login and also enabling TFO authentication
- Select the option for TFA and input the number to receive the OTP code via SMS
- Verify by input the code he/she has received via SMS
- Verification process done and directed to the next welcome pages and dashboard.
### what are the relevant pivot tracker stories?
- #184651063
